stacks-signer/src/signerdb.rs

```diff
@@ -80,15 +128,13 @@ impl SignerDb {
         let result: Option<String> = query_row(
             &self.db,
             "SELECT block_info FROM blocks WHERE reward_cycle = ? AND signer_signature_hash = ?",
-            &[&reward_cycle.to_string(), &format!("{}", hash)],
+            &[
```

Comment:
You can use `rusqlite::params!` to avoid the cast to `&dyn ToSql`

Reply:
`rusqlite::params!` just does `&dyn ToSql` under the hood https://docs.rs/rusqlite/latest/src/rusqlite/lib.rs.html#176-183. Given that it's not much more condense or easy to read, I would rather skip having a macro invocation here.

Reply:
I'm aware of what `rusqlite::params!` does. That's why I'm asking you to use it. We're in the process of switching the codebase over to it.

Reply:
Did not know we were in such a process. Changed now.